js/ui/form/ui.form.js

```diff
@@ -1181,7 +1181,7 @@ const Form = Widget.inherit({
 
             if(optionFullName === "formData") {
                 if(!isDefined(args.value)) {
-                    that._options.formData = args.value = {};
+                    that.option("formData", args.value = {});
```

Comment:
Should it be `that._setOptionByStealth`?
@jsdmitry  @dxvladislavvolkov 

@LazyLahtak Re-check similar code fragments, please

Reply:
You're right, I'll correct it.

Reply:
This code doesn't should lead to the optionChanged event

Reply:
> I'll correct it.

Do not forget to check the other code fragments (Switch, asyncTemplateMixin, SpeedDialButton etc)

Reply:
And we should use _getOptionByStealth instead of _option("optionName")

Reply:
We also have `_setOptionsSilent` (it works this). When we should use `_setOptionsSilent` and when `_setOptionByStealth`?

Reply:
I think we should replace `_setOptionsSilent` by `_setOptionByStealth`. It looks like API duplication.

Reply:
_setOptionsSilent doesn't call only  _optionChanged method https://github.com/DevExpress/DevExtreme/blob/20_1/js/core/component.js#L255

Reply:
Do we need `_setOptionsSilent` and `_setOptionByStealth` both? Can we keep one of scenario and get rid of other?

Reply:
@dxvladislavvolkov 
>And we should use _getOptionByStealth instead of _option("optionName")

I tried to use it (for example in `js/ui/speed_dial_action/*`) and it doesn't work (tests crashed)